.github/ISSUE_TEMPLATE/Epic.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Epic template
+about: Template for creating epics
+title: "[EPIC]"
+labels: Epic
+assignees: ''
+
+---
+
+<!--
+An Epic should define a higher-level package of work or project iteration that, once finished, makes sense to merge in core.
+-->
+## Description ##
+<!-- The what -->
+
+## Rationale ##
+<!-- The why -->
+
+## Stakeholders ##
```

Comment:
Should we have this? Will it be filled out?

Reply:
I'm not sure, probably not. On the one hand, it could _sometimes_ be useful to list them, but on the other, most stakeholders are going to be recurring (design and a11y team, Matías, etc.)